public/js/app.js

```diff
@@ -0,0 +1,84 @@
+let cart = new ShoppingList();
+let addItemButton = document.getElementById('addShoppingListItemButton');
+addItemButton.addEventListener('click', addToShoppingList);
+
+function addToShoppingList() {
+  let newShoppingListItem = new ShoppingListItem(title.value, description.value);
+  cart.addItem(newShoppingListItem);
+  let output = cart.render();
+  content.innerHTML = output;
+
+  let findCheckboxes = document.getElementsByClassName('checkbox');
+
+  for (let i = 0; i < findCheckboxes.length; i++) {
+    findCheckboxes[i].addEventListener('change', function() {
+      changeCheckedStatus(i, findCheckboxes[i]);
+    });
+  }
+  let findXButton = document.getElementsByClassName('xButton');
+  for (let i = 0; i < findXButton.length; i++) {
+    findXButton[i].addEventListener('click', function() {
+      removeItemButtonClicked(i);
+    });
+  }
+  for (let i = 0; i < cart.items.length; i++) {
+    if (cart.items[i].isDone === true) {
+      findCheckboxes[i].checked = true;
```

Comment:
Handling whether the checkboxes are checked or not should be handled in the render method for shopping list item.